doc/nrf/releases/release-notes-2.1.1.rst

```diff
@@ -0,0 +1,137 @@
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.1.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.1.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.1.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_211_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Matter
+------
+
+* Added:
+
+  * Feature-complete :ref:`production support <software_maturity>` for Matter over Thread.
+  * Documentation about :ref:`ug_matter_device_types`.
+
+* Updated documentation about Data Model and Interaction Model by moving it to separate pages: :ref:`ug_matter_overview_data_model` and :ref:`ug_matter_overview_int_model`, respectively.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``561d23d0db215a99705ff0696e73853c8edf11b2``.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Updated Matter repository to the official Matter 1.0 version tag.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+Thingy:53 Matter weather station
+--------------------------------
+
+* Added new factory data, PICS, and configuration overlay files.
+  These are useful for getting to know the Matter certification process.
+* Updated the documentation page with a new section about :ref:`matter_weather_station_cert_files`.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Matter samples
+--------------
+
+* Updated ZAP configuration of the samples to conform with device types defined in Matter 1.0 specification.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``71ef669ea4a73495b255f27024bcd5d542bf038c``, plus some |NCS| specific additions.
+This is the same commit ID as the one used for |NCS| :ref:`v2.1.0 <ncs_release_notes_210>`.
+
+For a complete list of |NCS| specific commits since v2.1.0, run the following command:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v3.1.99-ncs1
```

Comment:
@rlubos , please check.

Reply:
There were no changes in `sdk-zehpyr` since 2.1.1, the log will be empty. Do we still need to mention it?

Reply:
I'll rephrase, thanks.